### DISCUSSION TITLE
Wrappers for finding cells (along line etc.)

### DISCUSSION THREAD

**Discussion by leonfoks** [pyvista/pyvista#1667] [Ideas]
The VTK library has methods for finding cells along a line and I didn't see the corresponding wrappers in pyvista.
Is this something that will be implemented in the future?

**Comment by akaszynski** [comment]
Sure!  Please open a [feature request](https://github.com/pyvista/pyvista/issues/new?assignees=&labels=feature-request&template=feature-request.md&title=).

If possible, please include any class definitions or examples to implement it.

**Reply by tkoyama010** [reply]
I opend https://github.com/pyvista/pyvista/issues/1702 . @leonfoks Could you check this feature is what you want?
